internal/runner/job/conflict.go

```diff
@@ -49,7 +49,7 @@ func (r *Runner) checkForConflict(jobName string) error {
 	// if there is a job with this name, that has no meta, it was
 	// created by something other than the package manager and this
 	// process should fail.
-	if existing.Meta == nil {
+	if existing.Meta == nil && !r.cfg.RunConfig.DeployOverride{
```

Comment:
Rather than checking this flag multiple times, can't we check it once above all these checks and `return nil` if set?